components/event-service/internal/events/shared/constants.go

```diff
@@ -34,5 +34,12 @@ const (
 	FieldEventTime        = "event-time"
 	FieldEventType        = "event-type"
 	FieldEventTypeVersion = "event-type-version"
-	FieldData             = "data"
+
+	FieldEventIDV3          = "id"
+	FieldEventTimeV3        = "time"
+	FieldEventTypeV3        = "type"
+	FieldEventTypeVersionV3 = "eventtypeversion"
```

Comment:
The `eventtypeversion` is hard to read, please make it more readable.

Reply:
It follows the spec convention